admin/acceptance.py

```diff
@@ -964,6 +964,16 @@ def postOptions(self):
         provider = self['provider'].lower()
         provider_config = self['config'].get(provider, {})
 
+        if self.get('flocker-version') is not None:
+            try:
+                parse_version(self['flocker-version'])
+            except UnparseableVersion:
+                raise UsageError(
+                    "Flocker version {!r} is not a valid version".format(
+                        self['flocker-version']
+                    )
+                )
+
```

Comment:
:question: ``twisted.python.options.Options`` supports "Type Enforcement" 
 * https://twistedmatrix.com/documents/current/core/howto/options.html#type-enforcement
Consider moving this into a separate function which might be reusable elsewhere (I'd use it in my publish-installer-images script.) and which wouldn't need to handle the `None` case since that's a default value.